spec/system/admin/terms_of_service/distributions_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin TermsOfService Distributions' do
+  let(:user) { Fabricate(:admin_user) }
+  let(:terms_of_service) { Fabricate(:terms_of_service, notification_sent_at: nil) }
+
+  before { sign_in(user) }
+
+  describe 'Sending test TOS email', :inline_jobs do
+    it 'generates the test email' do
```

Comment:
This doesn't seem like what the spec is testing, it's testing sending the mail to all, isn't it?

Reply:
I probably copy/pasted this. Updated to more accurately reflect whats being done here.